.gitignore

```diff
@@ -21,3 +21,4 @@
 vendor/**/
 .env
 coverage.txt
+go.work.sum
```

Comment:
Matches this change from the Images Gen. 2 project branch: https://github.com/linode/linodego/blob/proj/images-gen2/.gitignore#L25